terminal_darwin.go

```diff
@@ -0,0 +1,52 @@
+// +build darwin
+//
+// Copyright (c) 2017 Intel Corporation
```

Comment:
I see that this is based on `terminal_linux.go` but I think you need to add an additional 2018 copyright with your name against it. Is that correct @grahamwhaley?

Reply:
correct - you may optionally want to add your own copyright line @raravena80 . But, don't update the **Intel** line to 2017-2018, if an Intel person didn't touch it ;-)

Reply:
ok cool, I just left it like it was :)